src/Administration/Resources/app/administration/src/app/store/license-violation.store.ts

```diff
@@ -2,6 +2,7 @@
  * @sw-package framework
  * @private
  */
+console.log('LOADED FILE LICENSE VIOLATION STORE');
```

Comment:
in this example console log is visible only when actual store is used, so to confirm it's loaded only when used